packages/sitecore-jss/src/media-api.ts

```diff
@@ -63,6 +64,10 @@ export const updateImageUrl = (
   params?: { [key: string]: string | number | undefined } | null,
   mediaUrlPrefix: RegExp = mediaUrlPrefixRegex
 ) => {
+  if (!params) {
```

Comment:
I'm wondering, should we check if `params` is empty object? For example using `Object.keys(params).length`? For example I didn't provide any properties, but anyway in this case url will be changed

Reply:
@illiakovalenko I went back and forth on this. Had that in there, but then found it was nice as it provides a way to "force" the prefix switch without any parameters (if desired). Useful in unit tests, possibly in the real world as well?

Reply:
@ambrauer But probably, use will define something or will not define anything

Reply:
@illiakovalenko I'm convinced, we should check object keys b/c of cases where an empty object is passed in by the frameworks usage (and in fact the srcset logic appears to do that). I'll make the change.

Reply:
@ambrauer I can't imagine case when I can specify empty object :)

Reply:
@ambrauer Cool, thank you

Reply:
Done